src/ripple/websocket/AutoSocket.h

```diff
@@ -207,61 +219,70 @@ class AutoSocket
     }
 
     template <typename Seq, typename Condition, typename Handler>
-    void async_read_until (const Seq& buffers, Condition condition, Handler handler)
+    void async_read_until(
+        const Seq& buffers, Condition condition, Handler handler)
     {
-        if (isSecure ())
-            boost::asio::async_read_until (*mSocket, buffers, condition, handler);
+        if (isSecure())
+            boost::asio::async_read_until(
+                *mSocket, buffers, condition, handler);
         else
-            boost::asio::async_read_until (PlainSocket (), buffers, condition, handler);
+            boost::asio::async_read_until(
+                PlainSocket (), buffers, condition, handler);
     }
 
     template <typename Allocator, typename Handler>
-    void async_read_until (boost::asio::basic_streambuf<Allocator>& buffers, std::string const& delim, Handler handler)
+    void async_read_until(boost::asio::basic_streambuf<Allocator>& buffers,
```

Comment:
We lost the space before the paren in many of these changes. I won't call each one out.